index.ts

```diff
@@ -0,0 +1,54 @@
+const prompt = require("prompt-sync")({ sigint: true });
+const upperValue: string | null = prompt("What value would you like FizzBuzz to go up to? ");
+console.log(`We go up to ${upperValue}`);
+
+function divisibleBy(num: number, divisor: number): boolean { return num % divisor == 0 };
```

Comment:
I know num and divisor are both of the same type but people tend to use triple equals instead of double equals anyway so I would change the double equals to a triple equals